layers/+completion/ivy/packages.el

```diff
@@ -219,15 +219,16 @@
   (define-key hydra-ivy/keymap [escape] 'hydra-ivy/keyboard-escape-quit-and-exit))
 
 (defun ivy/init-ivy-rich ()
-  (use-package ivy-rich
-    :defer t
-    :init
-    (progn
-      (setq ivy-rich-abbreviate-paths t
-            ivy-virtual-abbreviate 'full
-            ivy-rich-switch-buffer-align-virtual-buffer t)
-      (ivy-set-display-transformer 'ivy-switch-buffer
-                                   'ivy-rich-switch-buffer-transformer))))
+  (when ivy-enable-ivy-rich
```

Comment:
Sorry that I didn't use review system. Hopefully last comment!

You can just use something like this in the `(setq ivy-packages..` block to achieve this. Please test once.
```(ivy-rich :toggle ivy-enable-ivy-rich)```

Reply:
Yep, that works.